src/print.js

```diff
@@ -0,0 +1,32 @@
+import _ from 'lodash';
+import { input, todosUL } from './index.js';
+
+export default function addTodo(todo) {
+  let todoText = input.value
+  if (todo) {
+      todoText = todo.text
+  }
+  if (todoText) {
+      let todoEl =  document.createElement('li')
+      if (todo && todo.completed) {
+          todoEl.classList.add('completed')
+      }
+      todoEl.innerText = todoText
+      todosUL.innerHTML = `
+      <li class="todoEl">
+      <div>
+      <input class="checker" value="true" type="checkbox"/>
+      <label>
+      ${ todoEl.textContent }
+      </label>
+      </div>
+      <button class="positioner" type="button">
+      <span>&#8729;</span>
+      <span>&#8729;</span>
+      <span>&#8729;</span>
+      </button>
+      </li>`
+      input.value = ''
+  }
+ 
+}
```

Comment:
- [ ] In this milestone, we are supposed to implement this. Kindly, implement the following as listed. 

- You need an array of some simple to-do tasks (array of objects) with each object containing three keys - description [string], completed [bool], index: [number].

```javascript
const tasks = [
  {
    description: 'An example description in string',
    completed: false,
    index: 10,
  },
];
```
- [ ]  It has a function that iterates over the tasks array and populates the HTML list item element for each task.

- [ ] On page load, the task array is rendered dynamically to the DOM. The tasks should appear in order of the `index` values for each task.

Reply:
Ok, I solved the linters and make a default task appear in the DOMloaded event. I will appreciate your feedback.